Bug fixes and small changes

initialseed opened the pull request:
I've added a placeholder "prods" to items.gd that will need to be changed.

initialseed committed:
Fixed farm page aphrodisiac and prods amount

initialseed committed:
QOL: Always show when Sebastian order is ready

initialseed committed:
Mod apply warning fix: removed space at beginning of line

initialseed committed:
QOL: Keep mansion category open when upgrade panel is closed

initialseed committed:
Added "Removes vices." to description of amnesia potion

initialseed committed:
Display lust after farm calculations

initialseed committed:
Make snail breeding count as sex

This eliminates the "unquenchable lust" message in the global report

initialseed committed:
Fix contentment message on cattle screen

AricTriton commented:
If you're changing the Prods, I would change them to an item/weapon instead of putting them in the consumable potion class. That is very likely to break things.

You'll also need to add something (likely in backwardscompatibility) that will take the current number of prods and convert them to the same number of the new item to avoid players losing them all. Removing them entirely may break saves at this point, so it may be work leaving that line for now with a #TBK - Remove Later above it for us to remove in a version or two after saves have been well updated and the variable definitely isn't being used

initialseed committed:
Added prods back to farminventory
Added code to migrate prods from farminventory->globals.itemdict
Note: farminventory.prods are set to 0 after migration
Removed effect, toxicity and weight from prods item

initialseed commented:
Done but I think migrating prods from an old save wasn't necessary because _on_buy_farmitem in Mansion.gd uses globals.itemdict to update the amount so farminventory.prods is always 0 (unless I missed something).

Also, what does TBK stand for?

AricTriton commented:
> 

TBK stands for To Be Continued (misspelling intentional). It's commonly used by a lot of corporations, writers, etc to designate somewhere that isn't complete to be able to quickly search for the document to complete them later. You may be correct in that function, but just to be on the safe side we can test it with it then drop it after confirming that it won't break any saves.

Welcome to the community, by the way! Thanks for contributing!
